core/src/main/java/io/kestra/core/tasks/debugs/GetLog.java

```diff
@@ -0,0 +1,106 @@
+package io.kestra.core.tasks.debugs;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.executions.LogEntry;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.repositories.LogRepositoryInterface;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.FileSerde;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.event.Level;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static io.kestra.core.utils.Rethrow.throwConsumer;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Output execution logs in a file.",
+    description = "This task is useful to propagate your logs."
+)
+@Plugin(
+    examples = {
+        @Example(
+            code = {
+                "level: WARN",
+                "tasksId: " +
+                "   - \"previous-task-id\""
+            }
+        )
+    }
+)
+public class GetLog extends Task implements RunnableTask<GetLog.Output> {
+
+    @Schema(
+        title = "Filter on specific task(s)"
+    )
+    @PluginProperty(dynamic = true)
```

Comment:
I don't think it should be dynamic as task IDs are static inside the flow definition.